pkg/testutils/builder/defaultcr.go

```diff
@@ -0,0 +1,64 @@
+package builder
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+type DefaultCRBuilder struct {
```

Comment:
Struggling a bit with this name. Maybe CustomResourceBuilder? SampleCRBuilder? UnstructuredCRBuilder?

Reply:
maybe `ModuleCRBuilder`?

Reply:
Anything than default :)